src/DeviceInterface/InterfacePrototype.js

```diff
@@ -702,6 +706,11 @@ class InterfacePrototype {
     openManageCreditCards () {}
     openManageIdentities () {}
 
+    /**
+     * @returns {Promise<any>}
+     */
+    async startCredentialsImportFlow () {}
+
```

Comment:
If you can move this moved to the `CredentialsImport` class, this placeholder can also be removed.